NEWS.md

```diff
@@ -1,4 +1,6 @@
 # Version 1.0.9000
+* Renamed elements in the list object returned by `get_efficacy` function.
+* Renamed `getEff` function to `get_efficacy`.
```

Comment:
please keep original name `getEff` - otherwise we would have to change all camel case function names, which we can't as it breaks too much code. ok to rename some list elements here - generally again we need to be very careful for functions that are used more often like the models etc.